auth/tests/test_utils.py

```diff
@@ -21,6 +19,11 @@ def setUp(self):
     def test_no_header(self):
         utils.prepare(RequestFactory().get('/'))
 
+    def test_body(self):
+        post = RequestFactory().post('/', body='foo')
+        post.META[settings.HEADER_DESTINATION] = '/some/url/'
+        utils.prepare(post)['body'] = 'foo'
```

Comment:
did you mean ``eq_(utils.prepare(post)['body'], 'foo')`` ? Otherwise, I'm not sure what this is testing.